### PR TITLE
Update ReadME.md. Including Description on the functionality of our w…

### PR DESCRIPTION
**Pull request by pengxuemoua**
Include description on the functionality of our web app, which APIs are used and how to install and run our app.

This is done to resolve issue #5 

Sending to @Mr-3rd and @amylynndee to approve. 

FYI, I referenced Clara's Flask App readme: https://github.com/claraj/cat_flask/blob/master/README.md

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,26 @@
...
+## To install and run
+
+* Clone our repository at: Mr-3rd/MCTC_ITEC2905_fall2023_proj3_team03_Justin_Amy_Peng (github.com)
+* You will need to create your [Flickr API key]( https://www.flickr.com/services/developer/api/). 
+* You will need to create your [Yelp API key]( https://docs.developer.yelp.com/reference/v3_business_search). 
+* You will need to create your [YouTube API key](https://developers.google.com/youtube/registering_an_application). You will need just the API key, you can disregard the OAuth credentials.  
+* Create an environment variable **FLICKR_API ** holding your Flickr API key.
+* Create an environment variable **YELP_API ** holding your Yelp API key.
+* Create an environment variable **YOUTUBE_API ** holding your YouTube API key.
+* Create a Cat API key at https://thecatapi.com/signup and create an environment variable **CAT_API_KEY** holding your key
```
Comment (Mr-3rd) [inline README.md]: We can remove the lines that references cats.

**Review by Mr-3rd — CHANGES_REQUESTED** [review]
@pengxuemoua Let's remove all cat references and we can proceed

File: README.md (hunk 1 above)
Comment (pengxuemoua) [reply README.md]: I saw that too! I just removed it. @Mr-3rd

**Comment by pengxuemoua** [comment]
Per @Mr-3rd, we can proceed after removing cat references. Cat references removed.

Merging to Main.

**Pull request merged**